contents/tutorials/when-and-how-to-run-group-targeted-a-b-tests.md

```diff
@@ -0,0 +1,88 @@
+---
+title: When and how to run group-targeted A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-21
+featuredImage: ../images/tutorials/when-and-how-to-run-group-targeted-a-b-tests/user-vs-group-a-b-tests.png
```

Comment:
This unfortunately looks weird as preview image
![Screen Shot 2023-04-21 at 9 44 38 AM](https://user-images.githubusercontent.com/34755028/233690735-14e1eaa5-3589-4e55-989a-6001dbea57a3.png)